test/requirements.txt

```
@@ -0,0 +1,7 @@
+pytest==7.2.1
+pytest-ordering==0.6
+pytest-asyncio==0.20.3
+pytest-cov==4.0.0
+types-mock
+coverage2clover==3.3.0
+coveragepy-lcov==0.1.2
```

please check if we really need all the pytest-modules.
```suggestion
pytest==7.2.1
pytest-ordering==0.6
pytest-asyncio==0.20.3
pytest-cov==4.0.0
```